### ISSUE TITLE
Live Wizard working on Firefox but not Chrome

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#15401]
### Current behavior

**Original report from https://github.com/unoplatform/uno/pull/15400**

> I could only follow this guide using Firefox, not google chrome. I am on Debian 12
>
> https://platform.uno/docs/articles/create-an-app-vscode.html

*Reported by @JBrolin90*

### Works on UWP/WinUI

None

**Comment by Youssef1313** [comment]
@JBrolin90 If you open the browser's developer tools → console, are you seeing any errors?

**Comment by JBrolin90** [comment]
Thank you Youssef1313 for converting this into an issue. I can confirm that new.platform.uno indeed works on Chrome on my Windows 11 Home, Windows 11 Pro (under VirtualBox) and on my Linux Mint. But it still does not work on my Debian 12 machine (under VirtualBox). (Chrome Version 121.0.6167.160 (Official Build) (64-bit)) I added a screenshot showing what it looks like and with developer tools activated, showing an empty console, i.e., no error messages. I did restart the Debian machine but to no avail. I also tried with both NAT and Bridge network adapters on the virtual machine.
This Debian machine is freshly installed, just a week ago, with the purpose of learning Uno, Only installed Chrome, dotnet 8 and vscode. I did not tinker with anything. 

Would be great to understand what is going on. For now it is a small inconvenience as I can use FireFox as a workaround 

![Screenshot from 2024-02-12 06-54-45](https://github.com/unoplatform/uno/assets/51434679/bc268ef4-0f8c-4085-9799-d78b170d601c)

Thank you for your attention and let me know if I should provide some further details

Kind regards
Joachim

**Comment by jeromelaban** [comment]
@JBrolin90 thanks for the updates. In the console, there's a "Default levels" dropdown, can you select `Verbose` as well, refresh the app and tell us what you see?

**Comment by JBrolin90** [comment]
Yes, now something showed up

[ServiceWorker] Initializing
uno-bootstrap.js:699 Active service worker found, skipping register
Uno.UI.js:3716 Making persistent: /local
Uno.UI.js:3716 Making persistent: /roaming
Uno.UI.js:3716 Making persistent: /shared
setImmediate.js:111 [Violation] 'message' handler took 302ms

**Comment by jeromelaban** [comment]
Thanks. It's not showing enough information, still.

When testing with building a blank app (using a template from https://new.platform.uno), does the app start properly in this particular browser?

**Comment by JBrolin90** [comment]
Yes, I get the "Hello Uno platform!" message, exactly as under Windows

**Comment by jeromelaban** [comment]
Thanks. This is indeed, hiding some other issue, which may be related to webassembly features. Does https://calculator.platform.uno work?

Also could you try building the same hello app in release mode, using `InterpreterAndAOT` uncommented in the project file? It will be closer to the configuration of the wizard.

**Comment by JBrolin90** [comment]
You're welcome
Yes https://calculator.platform.uno/ works well.
I am now trying to run the release version, struggling a bit with all the options. Please hang on until I figure out how to issue a proper dotnet run command....

**Comment by JBrolin90** [comment]
Yes, it seems to be working as well. This is what I did (Not that hard actually)
$ cd ~/repos/Hello/Hello.Wasm
$ dotnet run --confuguration Release 

Then I opened localhost:5001 in Chrome and I got this after a few seconds:
![Screenshot from 2024-02-13 10-41-43](https://github.com/unoplatform/uno/assets/51434679/b3b77564-38c9-4c00-8078-6827023eb5cc)


Finally, again, I tried the new.platform.uno and it still does not work....

**Comment by JBrolin90** [comment]
This is my create command, for reference

dotnet new unoapp -o Hello -id "com.companyname.Hello" -pub "O=Hello" -markup "csharp" -platforms "wasm" -theme-service  -pwa False -theme "material" -presentation "mvux" -config false -di false -log "none" -nav "blank" -http false -loc false -server false -tests "none" -toolkit  -dsp false

**Comment by jeromelaban** [comment]
Thanks for the update. 

Could you visit this link: https://lemon-beach-09f7c130f.4.azurestaticapps.net/?loglevel=info, this will show more information in the browser debug console, and paste the result here? This is a prerelease version of the wizard.

**Comment by JBrolin90** [comment]
The app itself, still does not get past the startup banner. The debug console displays a few more lines

Registering service worker for /
Uno.UI.js:3716 Making persistent: /local
Uno.UI.js:3716 Making persistent: /roaming
Uno.UI.js:3716 Making persistent: /shared
setImmediate.js:111 [Violation] 'message' handler took 265ms
service-worker.js:4 [ServiceWorker] Initializing
uno-bootstrap.js:710 Service Worker Registered
service-worker.js:7 [ServiceWorker] Installing offline worker
service-worker.js:10 [ServiceWorker] Caching app binaries and content

**Comment by jeromelaban** [comment]
Thank you, the lets go a level deeper, can you run it using `https://lemon-beach-09f7c130f.4.azurestaticapps.net/?loglevel=debug` ? It is going to be lots more verbose, make sure to paste it as a text file

**Comment by JBrolin90** [comment]
Weird. It did not produce more output, only this:
Active service worker found, skipping register
Uno.UI.js:3716 Making persistent: /local
Uno.UI.js:3716 Making persistent: /roaming
Uno.UI.js:3716 Making persistent: /shared
setImmediate.js:111 [Violation] 'message' handler took 215ms

And then one issue:

Depreciated feature used, one source

<ol class="children expanded affected-resources" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; list-style-type: none; padding: 3px 0px 0px; position: relative; user-select: text; display: block; color: rgb(31, 31, 31); font-family: roboto, ubuntu, arial, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li jslog="TreeItem; parent: parentTreeItem" role="treeitem" class="parent expanded" aria-expanded="true" aria-posinset="1" aria-setsize="1" style="box-sizing: border-box; min-width: 0px; min-height: 16px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: flex-start; margin-top: 0px; padding: 2px 5px 0px; background: var(--sys-color-cdt-base-container);"><div class="affected-resource-label" style="box-sizing: border-box; min-width: 0px; min-height: 0px; font-size: 14px; line-height: 20px; color: var(--sys-color-on-surface); position: relative; cursor: pointer;">1 source</div></li><ol class="children expanded" role="group" style="box-sizing: border-box; min-width: 0px; min-height: 0px; list-style-type: none; padding: 6px 0px 9px 5px; display: block; background: var(--sys-color-cdt-base-container); margin-bottom: 10px;"><li jslog="TreeItem; parent: parentTreeItem" role="treeitem" style="box-sizing: border-box; min-width: 0px; min-height: 16px; text-overflow: ellipsis; white-space: normal; position: relative; display: flex; align-items: flex-start;"><span class="tree-element-title" style="box-sizing: border-box; min-width: 0px; min-height: 0px;"></span>
ai.2.6.3.min.js:5
--


</li></ol></ol>

**Comment by JBrolin90** [comment]
When I run that link on on Chrome under Windows, I do get verbose output though.

**Comment by ramezgerges** [comment]
This appear to also be happening on Ubuntu (both firefox and chrome as far as I've tried). The log on firefox gives this:
![image](https://github.com/unoplatform/uno/assets/66218781/f117d4f9-e204-4c6c-8628-045b69d3fddd)
The log on chrome is not meaningful.

**Comment by JBrolin90** [comment]
Took some days. I'm leaning towards a platform issue rather than an Uno issue. I moved my Debian 12 VM from my Windows 11 Home host to a new Linux Mint (LMDE 6) host. Now the project wizard works in the very same browser.. Moving back to the Windows host, the issue re-appear. I would be fine closing this issue. Potentially addressing it to VirtualBox instead.

In any case, thank you all for your kind support. I must say that I did learn a lot during these days
